File: jour_3/jour3.pas
program ProgramName;
var 
    longueur : integer;
    largeur : integer;
begin
  write('Entrez la largeur : ');
  readln(largeur);
  write('Entrez la longueur : ');
  readln(longueur);
  writeln('le perimetre du rectangle vaut : ', 2*(largeur+longueur));
  writeln('l''aire du rectangle vaut : ',largeur*longueur);

end.

File: jour_3/jour3.c
#include<stdio.h>
int main(){
    int largeur, longueur;
    printf("Entrez respectivement la largeur et la longeur : ");
    scanf("%d %d", &largeur, &longueur);
    printf("le perimetre de ce rectangle vaut : %d\n", (largeur + longueur)*2);
    printf("l'air de ce rectangle vaut : %d\n", largeur*longueur);
    return 0;
}